fs/vfs/fs_epoll.c

```diff
@@ -365,7 +365,7 @@ int epoll_ctl(int epfd, int op, int fd, struct epoll_event *ev)
         return -1;
     }
 
-  if (eph->poll[0].sem)
+  if (eph->poll[0].cbargs)
```

Comment:
remove the check because it's already checked by poll_notify

Reply:
Done